src/test/scala/mesosphere/marathon/core/task/tracker/impl/InstanceStateOpProcessorDelegateTest.scala

```diff
@@ -1,175 +0,0 @@
-package mesosphere.marathon
```

Comment:
I don't think you can remove this tests without testing the underlying functionality elsewhere. You'll have to convert tests to test the `InstanceTracker` then.

Reply:
Good point. Done.